webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/GroupsApi.scala

```diff
@@ -1358,20 +1359,21 @@ class GroupApiService14(
                                 s"Inherited properties for node '${nodeFact.id.value}' were not found while it's contained into group '${groupId.serialize}''"
                               )
                               .map { childProperties =>
-                                parentProperties.map { p =>
-                                  val matchingChildProperties = childProperties.collect {
-                                    case cp if cp.prop.name == p.prop.name => cp.hierarchy
-                                  }.toList
-
-                                  val hasConflicts = MergeNodeProperties
-                                    .checkValueTypes(matchingChildProperties.map(NodePropertyHierarchy(p.prop, _)))
-                                    .isLeft
-
-                                  (
-                                    p,
-                                    matchingChildProperties.flatten,
-                                    hasConflicts
-                                  )
+                                parentProperties.collect {
+                                  case p if p.prop.visibility == Displayed =>
```

Comment:
I wonder why this is not also done in the target JSON mapper/transformer (here to `JRGroupProperty`), as done with the properties for `JRGroup` elsewhere : 
```
.withFieldComputed(_.properties, _.properties.filter(_.visibility == Displayed).map(JRProperty.fromGroupProp(_)))
```

Reply:
I think I don't understand where you would want to put it ? I need to filter from the list of properties, not when I transform just one ?

Reply:
I am asking because in the case of `JRNodeDetailLevel` and `JRGroup` the filtering was done at the level of mapping to the JSON class for serialization. I expected it to be the same everywhere else because otherwise it would not guarantee that the JSON will not contain hidden ones if we just forget to filter somewhere before the final serialization step